wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/service/wahlvorstand/KonfigurierterWahltagClient.java

```diff
@@ -0,0 +1,8 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand;
+
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.WlsException;
+
+public interface KonfigurierterWahltagClient {
+
+    KonfigurierterWahltagModel getKonfigurierterWahltag() throws WlsException;
```

Comment:
So wie ich den Code sehe ist dem Service nur das Datum relevant. Die anderen Daten brauchen wir nicht.

Reply:
der WahlenClientImpl braucht die wahltag nummer, daher habe ich die nicht entfernt.

siehe commit [df52919](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/df5291992d3fbf41f9d7da791ba68dbe6fd29acd)